packages/server/src/tag/pipe/tag-list-query-pipe.spec.ts

```
@@ -0,0 +1,35 @@
+import { TagListQueryPipe } from './tag-list-query-pipe';
+import { TagListQueryDto } from '../dto/tag-list-query.dto';
+
+describe('TagListQueryPipe', () => {
+  let pipe: TagListQueryPipe;
+
+  beforeEach(() => {
+    pipe = new TagListQueryPipe();
+  });
+
+  it('should be defined', () => {
+    expect(pipe).toBeDefined();
+  });
+
+  describe('success calls', () => {
+    it('should let the TagListQueryDto go on through', () => {
+      const tagListQueryDto = new TagListQueryDto('1', 'javascript');
+      const result: TagListQueryDto = { page: 1, keyword: 'javascript' };
+      expect(pipe.transform(tagListQueryDto)).resolves.toBe(result);
+    });
+  });
+
+  describe('unsuccess calls', () => {
+    it('should throw an error if given a non-numberic page', () => {
```

`it should throw an error if non-numeric page is given`이라고 작성하면 더 좋을것 같습니다 ㅎㅎ